library/alloc/src/string.rs

```diff
@@ -492,7 +492,7 @@ impl String {
     /// `from_utf8_lossy()` will replace any invalid UTF-8 sequences with
     /// [`U+FFFD REPLACEMENT CHARACTER`][U+FFFD], which looks like this: �
     ///
-    /// [byteslice]: ../../std/primitive.slice.html
+    /// [byteslice]: core::slice
```

Comment:
This now links to the module, not the primitive. Linking to slice with intra-doc links isn't currently possible: https://github.com/rust-lang/rust/issues/63351
```suggestion
    /// [byteslice]: ../../std/primitive.slice.html
```